### DISCUSSION TITLE
My Start with Foam

### DISCUSSION THREAD

**Discussion by J450NDR02D** [foambubble/foam#1352] [Show and tell]
Hi, my name is Jason R Drozd, I am an aircraft mechanic. For some time now I have been looking for a way to capture and keep track of information with work and personal and ideas I might have or come across as I go about my day at work and home. My system so far is a work in progress and I refine it a little at a time as I go forward with it. I am self-taught and learning as I go.
I started trying to come up with a system a couple of years ago and started with org-mode in Emacs. This was ok for a while until the configuration started to become difficult to get to work with want I wanted it to do. I even tried to get org-roam to work and the configuration problems were the same, if not more, than org-mode, and the process to input context was cumbersome to use for me. Next I ran across Obsidian. This one was easier to use, but I did not like the folder base system, I found it hard to place notes. Next was Logseq, which is like Obsidian but without the folders, all your notes are individual markdown files like Obsidian but put into one directory. I like the PDF plugin that lets you view and highlight PDF's side by side with your note file and allow you to link to your highlights and make notes about each highlight. I also like all the file in one directory and the ability to link to any other file but any highlighted text that you might copy into your note from the web or another source create a separate file in your directory automatically if you wanted it to or not. While you can do a search for what you might be looking for, it comes back with a long list of files that has your search term in it and more than one result referred to the same file. Also, for me, I did not like the look and feel of the interface. So all this brings me to Foam. I found Foam by doing a search for something dealing with VS Code and started to try it out about two weeks ago. While it does not have the PDF highlighting as in Logseq, I like the one directory for all my note files, the links are not creating a file for every link I might copy in on it own. Also, I like the simplistic look and feel of the interface, I do not use VS Code for coding too much, but I do know how to use it for the most part.

Now I am still working on finding my way, workflow, and it is rough around the edges but here it is:

I am using the inbox file to capture my fleeting notes, work notes, and other things that pop up throughout the day that sound interesting, or I need to make a note of. At the beginning of the file is a list of things I need or want to do at the beginning of each day. The notes that I capture in the inbox file will be reviewed and put in their proper file, existing or new, and the useless ones will be deleted. I have a template for a daily note, and I am trying it out to keep track of what I do each day if need be.

I am trying to figure out how to deal with PDF's. I have the plugin to view PDF's in VS Code, but like I said, you can not highlight text in them or link to the highlighted parts. This is the part I am still working out and have not come up with a good workflow for yet.

I tried Zotero and Mendeley for a while and did not like their interface and the way Zotero's limited interaction with Foam and there is no plugin for Mendeley. I am now trying Docear, and it seems to be OK, but time will tell. In Docear you can highlight text, copy the highlights, make notes, and save the citation in BibTeX, but there is no plugin for Foam. I need to figure out how to do citations in Foam from the PDFs I am reading and making note from.

All in all I am liking Foam, and it seems to be working out so far for me, and I am working on my workflow in it. I am controlling my links better and the number of files being created is down to a minimum which I like. I have all the recommended plugins and a few more, and they seem to be working good. Like I said a few times already, time will tell.

Thanks to the Foam team for putting this together.

**Comment by riccardoferretti** [comment]
Hi Jason, thanks for sharing your experience with various tools, Foam, and how you are developing your system with it.
Please keep us posted :)
